src/content/api/asset-programs.mdoc

```diff
@@ -0,0 +1,92 @@
+---
+title: Asset Programs
+description: Asset Programs that Merchants can accept
+nav:
+  path: Assets
+  order: 4
+---
+
+An Asset Program is a container for one or more [Asset Types](/api/asset-types).
+
+## Asset Program Model
+
+{% properties %}
+  {% property name="id" type="string" %}
+    The unique identifier of the Asset Program.
+  {% /property %}
+
+  {% property name="name" type="string" %}
+    The display name of the Asset Program.
+  {% /property %}
+
+  {% property name="assetTypes" type="array" %}
+    One or more [Asset Types](/api/asset-types) that can be accepted as part of the Asset Program.
+  {% /property %}
+
+  {% property name="accountId" type="string" %}
+    The [Account](/api/accounts/) that owns the Asset Program.
+  {% /property %}
+
+  {% property name="test" type="boolean" %}
+    `true` if the Asset Program is for testing purposes only.
```

Comment:
Suggestion: We should mention this is based on the liveness of the account creating the asset program